admin/sql.usage.example.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const TYPES = require('tedious').TYPES
+const sqlConnectionService = require('./services/data-access/sql.connection.service')
+const sqlService = require('./services/data-access/sql.service')
+
+sqlConnectionService.init()
+
+const examples = {}
+
+examples.getData = () => {
+  sqlService.query('SELECT * FROM settings')
+    .then(result => console.log('query settings table result:', result))
```

Comment:
I think we need an example using `await` rather than `Promise.then` please.  That's our usual idiom.